src/config.rs

```diff
@@ -55,23 +55,23 @@ impl EgressSettings {
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct ScalingSettings {
-    pub desired_replicas: u32,
+    pub desired_replicas: Option<u32>,
```

Comment:
I don't think this should be optional. Supporting an empty scaling table in the toml seems unecessary